docs/src/guides/drupal9/redis.md

```diff
@@ -140,33 +137,19 @@ To configure your Redis service:
 
 ## Verify Redis is running
 
-To verify that Redis is running:
-
-1. [Connect to your app with SSH](../../development/ssh/). 
-
-2. In your SSH terminal, run the following command:
-
-   ```bash
-   echo $PLATFORM_RELATIONSHIPS | base64 --decode | json_pp
-   ```
-   In the output, retrieve the value of the `host` property for your Redis relationship.
-
-3. Run the following command:
+To verify that Redis is running, run the following command:
 
-   ```bash
-   redis-cli -h {{< variable "HOST" >}} info
-   ```
+```bash
+platform ssh 'echo $PLATFORM_RELATIONSHIPS | base64 --decode | json_pp'
+```
 
-   The output produces information and statistics about Redis,
-   showing that the service is up and running.
+In the output, retrieve the value of the `host` property for your Redis relationship.
 
-## Clear SQL cache tables
```

Comment:
About the SQL section, I did wonder about it too. It does seem out of scope and it's not critical information. It has more to do with optimization and doesn't seem to apply to all use case scenarios. I considered adding it as an optional step, but I think we could even remove it for now (which I have done in this draft, but can roll back if you think it's important input after all), and put it back in later if we get feedback that this kind of information is missing?